PeerioTechnologies/peerio-mobile
# fix: deleted user login error

#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/7272/mobile-incorrect-error-when-trying-to-sign-into-a-suspended-account

#### Testing instructions  
1. delete an existing user
1. in email confirmation, confirm deletion of that account
1. attempt to log in again using that username
1. "Account deleted" error should show now, instead of incorrect password error

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
